en/tokovoip.md

```diff
@@ -0,0 +1,155 @@
+---
+id: tokovoip
+title: Install TokoVOIP
+sidebar_label: Install TokoVOIP
+---
+
+> IMPORTANT: A **seperate Teamspeak Server is required for TokoVOIP to work.** You can book a Teamspeak Server from ZAP directly [here](https://zap-hosting.com/en/shop/product/teamspeak3-server/)(clickable)!
+
+## 📶 Connect to FTP
+
+Before TokoVOIP can be installed, you must be connected via [FTP](gameserver_ftpaccess.md).
+
+After you have succesfully connected via FTP, you can connect and open the following server folder.
+`/g#####/gta5-fivem/server-data/resources`
+
+## ⬇️ Prerequisites
+
+### Player Install 
+
+Every player must install Teamspeak 3 and the TokoVOIP plugin on their personal computer.
+
+For this tutorial, it's assumed you and your players have already installed the plugin on your local teamspeak client.
+
+You can find the guide for this [here](https://github.com/Itokoyamato/TokoVOIP_TS3#players-setting-up-the-ts3-plugin).
+
+## ▶️ TokoVOIP Server Install
+
+> In this guide for the FTP Application, we are using WinSCP. You are welcome to use the FTP Application of your choice! 
+
+### Installing/Configuring ws-server
+
+Firstly, you will need to download [ws_server](https://github.com/Itokoyamato/TokoVOIP_TS3/releases) from the TokoVOIP Github. 
+
+![](https://screensaver01.zap-hosting.com/index.php/s/zecMzwqWJbXXws7/preview)
+
+You will then simply drag it over to your `resources` folder via FTP.
+
+![](https://screensaver01.zap-hosting.com/index.php/s/HT2pDJDxA2GkWjf/preview)
+
+Then open the directory `ws_server` that you have uploaded to the servers FTP.
+
+Right Click, and then Edit the file  `config.js`
+
+![](https://screensaver01.zap-hosting.com/index.php/s/tPgiQBgjTkRKAC2/preview)
+
+In this file, we are only modifying the **TSServer** and the **WSServerPort**.
+
+Replace TSServer with your teamspeak servers **IPV4 ADDRESS**
+
+```
+TSServer = Your teamspeak IP Address (DO NOT INCLUDE THE PORT)
+WSServerPort = CAN BE ANYTHING ABOVE 30000 (DO NOT MAKE YOUR TS3 PORT) 
```

Comment:
the WSServerPort *must* be one of your available ports of the gameserver, random ports WILL cause technical issues.